.github/actions/e2e-deploy-vald/action.yaml

```diff
@@ -125,9 +125,7 @@ runs:
         echo "POD_NAME=${podname}" >> $GITHUB_OUTPUT
       env:
         VALUES: ${{ inputs.values }}
-        # TODO: currently, we can't pass helm options to the make command.
-        #       Is it really necessary in the first place?
-        # HELM_EXTRA_OPTIONS: ${{ inputs.helm_extra_options }}
```

Comment:
The "make k8s/vald/deploy" command is useful for building a Vald Cluster using the "Local Chart".

However, since the internal settings can only be changed statically by modifying values.yaml, I think it would be useful for CI if dynamic settings such as the number of vector dimensions could be changed on the command line.

For this reason, I believe the HELM_EXTRA_OPTS will be a very useful feature when using Local Charts in CI.